packages/zend-pdf/library/Zend/Pdf/Filter/RunLength.php

```diff
@@ -44,17 +44,30 @@ public static function encode($data, $params = null)
     {
         $output = '';
 
+        foreach (str_split($data, 128) as $chunk) {
+            $output .= self::chunkEncode($chunk);
+        }
```

Comment:
could you add a comment here, why the $data has to be split into chunks now, please?

Reply:
To work around this [bug](https://github.com/php/php-src/commit/d776d25a8e6763bfe8cc283a4693d7d417d50ddd). Which isn't in the 8.0.0 release.

Reply:
meanwhile travis is still fighting with setting up the 8.0 image https://travis-ci.community/t/php-8-0-missing/10132/14